doc/user/walkthrough/etcd-operator.md

```diff
@@ -0,0 +1,326 @@
+# etcd operator
+
+## Overview
+
+The etcd operator manages etcd clusters deployed to [Kubernetes][k8s-home] and automates tasks related to operating an etcd cluster.
+
+- [Create and destroy](#create-and-destroy-an-etcd-cluster)
+- [Resize](#resize-an-etcd-cluster)
+- [Recover a member](#member-recovery)
+- [Backup and restore a cluster](#disaster-recovery)
+- [Rolling upgrade](#upgrade-an-etcd-cluster)
+
+## Demo
+
+![etcd Operator demo](https://raw.githubusercontent.com/coreos/etcd-operator/master/doc/gif/demo.gif)
+
+## Getting Started
+
+> NOTE: This section assumes working directory is the etcd operator repo root.
+
+### Deploy etcd operator
+
+See [instructions on how to install/uninstall etcd operator](../install_guide.md) .
+
+### Create and destroy an etcd cluster
+
+```bash
+$ kubectl create -f example/example-etcd-cluster.yaml
+```
+
+A 3 member etcd cluster will be created.
+
+```bash
+$ kubectl get pods
+NAME                            READY     STATUS    RESTARTS   AGE
+example-etcd-cluster-0000       1/1       Running   0          1m
+example-etcd-cluster-0001       1/1       Running   0          1m
+example-etcd-cluster-0002       1/1       Running   0          1m
+```
+
+See [client service](../client_service.md) for how to access etcd clusters created by operator.
+
+If you are working with [minikube locally](https://github.com/kubernetes/minikube#minikube) create a nodePort service and test out that etcd is responding:
+
+```bash
+$ kubectl create -f example/example-etcd-cluster-nodeport-service.json
+$ export ETCDCTL_API=3
+$ export ETCDCTL_ENDPOINTS=$(minikube service example-etcd-cluster-client-service --url)
+$ etcdctl put foo bar
+```
+
+Destroy etcd cluster:
+
+```bash
+$ kubectl delete -f example/example-etcd-cluster.yaml
+```
+
+### Resize an etcd cluster
+
+Create an etcd cluster:
+
+```
+$ kubectl apply -f example/example-etcd-cluster.yaml
+```
+
+In `example/example-etcd-cluster.yaml` the initial cluster size is 3.
+Modify the file and change `size` from 3 to 5.
+
+```
+$ cat example/example-etcd-cluster.yaml
+apiVersion: "etcd.database.coreos.com/v1beta2"
+kind: "EtcdCluster"
+metadata:
+  name: "example-etcd-cluster"
+spec:
+  size: 5
+  version: "3.1.8"
+```
+
+Apply the size change to the cluster CR:
+```
+$ kubectl apply -f example/example-etcd-cluster.yaml
+```
+The etcd cluster will scale to 5 members (5 pods):
+```
+$ kubectl get pods
+NAME                            READY     STATUS    RESTARTS   AGE
+example-etcd-cluster-0000       1/1       Running   0          1m
+example-etcd-cluster-0001       1/1       Running   0          1m
+example-etcd-cluster-0002       1/1       Running   0          1m
+example-etcd-cluster-0003       1/1       Running   0          1m
+example-etcd-cluster-0004       1/1       Running   0          1m
+```
+
+Similarly we can decrease the size of cluster from 5 back to 3 by changing the size field again and reapplying the change.
```

Comment:
`Similarly we can decrease the size of cluster from 5 back to 3 by changing the size field again and reapplying the change`
->
`Similarly changing the field `size` to 3 from 5 also decreases the size of cluster to 3:`